styles/homepage.css

```diff
@@ -0,0 +1,37 @@
+.header{
+    padding-top: 2rem;
+    font-weight: 600;
+}
+
+.main-content{
```

Comment:
suggestion add a space before { use beautify extension in vs code

Reply:
thanks, will take care of it